### PR TITLE
added a method to get a list of all playlists for the current user

### PR DESCRIPTION
**Pull request by skoop**
I'm currently working on an application using this library but needed to add a method that gets all playlists for a listing of playlists. Here it is!

### CODE REVIEW

File: src/DeezerAPI.php [hunk 1]
```diff
@@ -111,6 +111,21 @@ public function getPlaylist($playlistId)
     }
 
     /**
+     * Get all playlists for the authenticated user
+     * 
+     * @return array
+     */
+    public function getPlaylists()
```
Comment (clemfromspace) [inline src/DeezerAPI.php]: Nipick: Since we got a `createUserPlaylist` method, maybe name this `getUserPlaylists` ?

**Comment by clemfromspace** [comment]
First of all, thanks for your contribution. Just a small nitpick and I will merge this :)

File: src/DeezerAPI.php (hunk 1 above)
Comment (skoop) [inline src/DeezerAPI.php]: Good point, will fix

**Comment by skoop** [comment]
I might send more pull requests soon, I'm going to be working on an application that will integrate deezer ;)

**Pull request merged**